docker-compose/7.1.N-docker-compose.yml

```diff
@@ -59,6 +59,16 @@ services:
         -XX:MinRAMPercentage=50
         -XX:MaxRAMPercentage=80
         -XX:MaxRAM=1900m
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.alfresco.rule=PathPrefix(`/`)"
+      - "traefik.http.services.alfresco.loadbalancer.server.port=8080"
+      - "traefik.http.routers.solrapideny.rule=PathRegexp(`^/alfresco/(wc)?s(ervice)?/api/solr/`)"
+      - "traefik.http.middlewares.fakeauth.basicauth.users=test:"
+      - "traefik.http.routers.solrapideny.middlewares=fakeauth@docker"
+      - "traefik.http.routers.alfrescomicrometer.rule=PathRegexp(`^/alfresco/(wc)?s(ervice)?/prometheus`)"
+      - "traefik.http.middlewares.prometheusipfilter.ipallowlist.sourcerange=127.0.0.0/8"
```

Comment:
it is sensible that prometheus source ip filter is set to localhost in a container environment?

Reply:
admitedly not but I can't come up with a sensible default as it's rather random. I see it as a placeholder for someone to easily update it so that the real prometheus IP can be set